modules/amq-custom-config/added/configure_custom_config.sh

```diff
@@ -11,6 +11,8 @@ function swapVars() {
   sed -i "s/\${AMQ_NAME}/$AMQ_NAME/g" $1
   sed -i "s/\${AMQ_ROLE}/$AMQ_ROLE/g" $1
   sed -i "s/\${AMQ_STORAGE_USAGE_LIMIT}/$AMQ_STORAGE_USAGE_LIMIT/g" $1
+  sed -i "s/\${AMQ_CLUSTER_USER/$AMQ_CLUSTER_USER/g" $1
```

Comment:
```suggestion
  sed -i "s/\${AMQ_CLUSTER_USER}/$AMQ_CLUSTER_USER/g" $1
```
? Haven't tested this directly but I suspect that the right brace is required.

Reply:
@RoddieKieley You are right. I missed it. Adding suggested changes. Thank you.